Doc/library/dis.rst

```diff
@@ -210,6 +214,11 @@ operation is being performed, so the intermediate analysis object isn't useful:
    This generator function uses the ``co_firstlineno`` and ``co_lnotab``
    attributes of the code object *code* to find the offsets which are starts of
    lines in the source code.  They are generated as ``(offset, lineno)`` pairs.
+   See :source:`Objects/lnotab_notes.txt` for the ``co_lnotab`` format and
+   how to decode it.
+
+   .. versionchanged:: 3.6
+      Line numbers can be not monotonically increasing.
```

Comment:
This is the wording for describing the implementation details of `co_lnotab`. What is important for the user `findlinestarts()` which shouldn't know anything about `co_lnotab` is that now this function can produce non-monotonic line numbers. That can affect user code.